oauthenticator/tests/test_azuread.py

```diff
@@ -33,7 +34,7 @@ def user_model(tenant_id, client_id, name):
             "iat": now,
             "nbf": now,
             "name": name,
-            "preferred_username": name,
+            "preferred_username": "preffered",
```

Comment:
A typo here, should be "preferred", but I don't understand the change from `name` to `"preferred"`.

Reply:
I just wanted to generate another different test. To have preffered be its own thing. But we can leave it like before if it's confusing

Reply:
Ah well its a mocked response, I think it will be `name` consistently, but we should reasonably respect the `preferred_username` rather than pick a username based on `name`, so good point about testing something different in this mocked response!